applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -174,7 +176,12 @@ def Initialize(self):
     def InitializeSolutionStep(self):
         if self._TimeBufferIsInitialized():
             (self.bdf_process).Execute()                    # Recompute the BDF2 coefficients
-            (self.level_set_convection_process).Execute()   # Convect the level-set according to the previous step velocity
+
+            if not self._bfecc_convection:
+                (self.level_set_convection_process).Execute()   # Convect the level-set according to the previous step velocity
+            else:
+                (self.level_set_convection_process).BFECCconvect(self.main_model_part, KratosMultiphysics.DISTANCE, KratosMultiphysics.VELOCITY, 10)
```

Comment:
What is the 10? For this kind of arguments it is a good practice to use an auxiliary local variable that has a descriptive name so any developer reading the code does not need to open the function (in this case the BFECCconvect, to check what does this argument represent. The overhead of this is minimum since the variable will be de-allocated as soon as you leave the scope of the function.

Reply:
Added to the .json settings